src/test/scala/com/lambtors/poker_api/module/poker/behaviour/PokerBehaviourSpec.scala

```diff
@@ -15,11 +14,11 @@ trait PokerBehaviourSpec
     with TypeCheckedTripleEquals
     with MockFactory
     with OneInstancePerTest
-    with ScalaFutures
-    with MockedPokerGameRepository
+    with ScalaFutures {
+  implicit val ec = ExecutionContext.global
 
-trait MockedPokerGameRepository extends MockFactory {
```

Comment:
why? 😢 , I think it's better with a trait per mocked instance so it's more reusable and cleaner

Reply:
I thought it wasn't necessary. 

If you want, if you have another instance, add it again :D